src/days-2-4-adv/adv.py

```diff
@@ -1,29 +1,39 @@
 from room import Room
 from player import Player
+from item import Item
 
-# Declare all the rooms
-
+# Declare all the rooms_______ Dictionary__________
+#We have created an "obj" of "objs" for the Rooms Class (room.py) , it is 'here' in adv.py b/c
+#the classes are just blue prints, this here below, is "The Room" content
 room = {
     'outside':  Room("Outside Cave Entrance",
-                     "North of you, the cave mount beckons"),
+                     "North of you, the cave mount beckons", "skull"),
 
     'foyer':    Room("Foyer", """Dim light filters in from the south. Dusty
-passages run north and east."""),
+passages run north and east.""", "pickle"),
 
     'overlook': Room("Grand Overlook", """A steep cliff appears before you, falling
 into the darkness. Ahead to the north, a light flickers in
-the distance, but there is no way across the chasm."""),
+the distance, but there is no way across the chasm.""", "trash puppy"),
 
     'narrow':   Room("Narrow Passage", """The narrow passage bends here from west
-to north. The smell of gold permeates the air."""),
+to north. The smell of gold permeates the air.""", "boot"),
 
     'treasure': Room("Treasure Chamber", """You've found the long-lost treasure
 chamber! Sadly, it has already been completely emptied by
-earlier adventurers. The only exit is to the south."""),
+earlier adventurers. The only exit is to the south.""", "toothpick butterfly"),
 }
 
-# Link rooms together
+#__________Item Dictionary___________
+items = {"pickle": "pickle", "skull": "skull", "boot": "boot", "trash": "trash","puppy": "puppy",
```

Comment:
Remember that the items in this game are objects instantiated with the Item class, so consider setting up this Item Dictionary similar to how the rooms dictionary was set up.